### PR TITLE
Added board config for OAK-D-W (DM9098 R7M2E7)

### PR DESCRIPTION
**Pull request by ahrastnik**

### CODE REVIEW

File: batch/eeprom/DM9098_R7M2E7_oak_d_w.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "batchName": "Maxwell",
+    "batchTime": 0,
+    "boardConf": "nIR-C00M00-00",
```
Comment (Luxonis-David) [inline batch/eeprom/DM9098_R7M2E7_oak_d_w.json:4]:
```suggestion
    "boardConf": "nIR-C21M05-00",
```

Comment (Luxonis-David) [reply batch/eeprom/DM9098_R7M2E7_oak_d_w.json:4]: Please note that this configuration is for OAK-D-W not for OAK-D-W-97

File: batch/oak_d_w.json [hunk 2]
```diff
@@ -5,6 +5,12 @@
     "test_type": "OAK-D",
     "options": {"bootloader": "header_usb", "imu": true},
     "variants": [
+        {
+            "title":"OAK-D W (DM9098 R7M2E7)",
+            "description": "OAK-D W based on DM9098 R7M2E7 board, with wide lens",
+            "eeprom":"eeprom/DM9098_R7M2E7_oak_d_w.json",
+            "board_config_file": "OAK-D-W.json"
```
Comment (Luxonis-David) [inline batch/oak_d_w.json:12]: Should it be OAK-D-W-97 instead?

**Review by Luxonis-David — CHANGES_REQUESTED** [review]

File: batch/oak_d_w.json (hunk 2 above)
Comment (ahrastnik) [reply batch/oak_d_w.json:12]: Ok, yes, it was a misunderstanding, this should indeed be the OAK-D-W-97. However, the config for that board already exists. I'm waiting for Andy to clear it up.

**Comment by ahrastnik** [comment]
Closing this, as this was a misunderstanding. The correct config was already presenet, but the flashing process failed and it was mistook for a non-existent config.

**Pull request closed without merging**